_common/core_ref.md

```diff
@@ -1,5 +1,21 @@
 ---
-layout: externalurl
-redirect_url: https://core.blockstack.org/
+layout: core
 title: "Stacks Node API"
+description: Interacting with the Blockstack Blockchain
+permalink: /:collection/:path.html
 ---
+
+With the launch of Stacks 2.0, a new version of the Blockstack blockchain was released. You currently have two ways of interacting with the blockchain:
+
+- Using a hosted HTTPS REST client that makes RPC calls to the blockchain
```

Comment:
I have a slightly different understanding of what a REST client is. I'd consider this to be something like Postman. 

I'd describe the sidecar here: An API we use to address performance issues for which querying a node itself would be too slow/painful. And that, for convenience, it has a proxy to the core node.